src/wp-includes/interactivity-api/class-wp-interactivity-api.php

```diff
@@ -973,29 +994,34 @@ private function get_router_animation_styles(): string {
 	}
 
 	/**
-	 * Outputs the markup for the top loading indicator and the screen reader
-	 * notifications during client-side navigations.
-	 *
-	 * This method prints a div element representing a loading bar visible during
-	 * navigation, as well as an aria-live region that can be read by screen
-	 * readers to announce navigation status.
+	 * Deprecated.
 	 *
 	 * @since 6.5.0
+	 * @deprecated 6.7.0 Use {@see WP_Interactivity_API::print_router_markup} instead.
 	 */
 	public function print_router_loading_and_screen_reader_markup() {
+		_deprecated_function( __METHOD__, '6.7.0', 'WP_Interactivity_API::print_router_markup' );
+
+		// Call the new method.
+		$this->print_router_markup();
+	}
```

Comment:
I believe we need to call the "new" method here. I've just added it in https://github.com/WordPress/wordpress-develop/pull/7447/commits/e09839e079b75bd32e9bab0aad0502c0ee333886